docker-compose.yaml

```diff
@@ -207,6 +208,15 @@ services:
       - MICRONAUT_ENVIRONMENTS=${CRON_MICRONAUT_ENVIRONMENTS}
     volumes:
       - workspace:${WORKSPACE_ROOT}
+  airbyte-connector-builder:
+    image: airbyte/connector-builder:${VERSION}
+    logging: *default-logging
+    container_name: airbyte-connector-builder
+    restart: unless-stopped
+    ports:
+      - 8080:80
```

Comment:
Based on a comment in another PR, we may want to choose something less common than 8080 here? Maybe something like 8003 (8001 is used for airbyte-server, 8002 is used for airbyte-cloud-server -- I think those are the only ones used currently based on quick searching for `8003` in our codebases but not 100% sure)

Reply:
the airbyte-proxy is the only application with a localport today after https://github.com/airbytehq/airbyte/pull/17694. we should make sure this isn't exposed to localhost by default

Reply:
[done](https://github.com/airbytehq/airbyte/pull/18410/commits/71cae24b3a2b156ccb3b8d2a434006089b556b8b) and confirmed the port is not reachable from localhost, but the FE can still reach the endpoint